packages/synapse-react-client/src/components/SynapseSearchPageResults/SynapseSearchResultsCard.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Box,
+  Paper,
+  PaperProps,
+  Button,
+  styled,
+  Typography,
+  Chip,
+} from '@mui/material'
+import { StyledComponent } from '@emotion/styled'
+import FavoriteButton from '../favorites/FavoriteButton'
+import {
+  Update as UpdateIcon,
+  Download as DownloadIcon,
+} from '@mui/icons-material'
+
+export type SynapseSearchResultsCardProps = {
+  name: string
+  entityType: string
+  modifiedOn: Date
+}
+
+const SynapseSearchResultsCardContainer: StyledComponent<PaperProps> = styled(
+  Paper,
+  {
+    label: 'SynapseSearchResultsCardContainer',
+  },
+)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  minHeight: '250px',
+  width: '1000px',
```

Comment:
This width should not be hard-coded, it will need to adapt to the size of the container it's in (which will vary by screen size)
```suggestion
  width: '100%',
```